src/components/@shared/FormFields/FilesInput/index.tsx

```diff
@@ -7,55 +7,30 @@ import { InputProps } from '@shared/FormInput'
 import { initialValues } from 'src/components/Publish/_constants'
 import { getFileUrlInfo } from '@utils/provider'
 import { FormPublishData } from 'src/components/Publish/_types'
+import { LoggerInstance } from '@oceanprotocol/lib'
 
 export default function FilesInput(props: InputProps): ReactElement {
   const [field, meta, helpers] = useField(props.name)
   const [isInvalidUrl, setIsInvalidUrl] = useState(false)
   const [isLoading, setIsLoading] = useState(false)
   const { values } = useFormikContext<FormPublishData>()
 
-  const loadFileInfo = useCallback(
-    (url: string) => {
-      const providerUri =
-        (values.services && values.services[0].providerUrl.url) ||
-        'https://provider.mainnet.oceanprotocol.com'
-
-      async function validateUrl() {
-        try {
-          setIsLoading(true)
-          const checkedFile = await getFileUrlInfo(url, providerUri)
-          setIsInvalidUrl(!checkedFile[0].valid)
-          checkedFile && helpers.setValue([{ url, ...checkedFile[0] }])
-        } catch (error) {
-          toast.error(
-            'Could not fetch file info. Please check URL and try again'
-          )
-          console.error(error.message)
-        } finally {
-          setIsLoading(false)
-        }
-      }
-
-      validateUrl()
-    },
-    [helpers, values.services]
-  )
-
-  useEffect(() => {
-    // try load from initial values, kinda hacky but it works
```

Comment:
if I got this right, we only had this effect for refilling this field from saved `localStorage` form values but we don't have this right now so removed it all. If anybody remembers another reason we had this, let me know